apps/next/src/content/docs/llamaindex/modules/workflows.mdx

```diff
@@ -116,7 +116,7 @@ const jokeFlow = new Workflow({ verbose: true, validate: true });
 Optionally, you can choose to use global context between steps. For example, maybe multiple steps access the original `query` input from the user. You can store this in global context so that every step has access.
 
 ```typescript
-import { Context } from "@llamaindex/core/workflow";
+import { Context } from "llamaindex";
```

Comment:
here we can use the workflow package?
```suggestion
import { Context } from "@llamaindex/workflow";
```

Reply:
ah now i see the problem, this is an outdated document using the old workflow format - update it to use the new workflows like in https://ts.llamaindex.ai/docs/llamaindex/guide/workflow